lib/client.js

```diff
@@ -9,20 +9,20 @@ const EventEmitter = require('events').EventEmitter;
 const ValidationError = utils.LimitdRedisValidationError;
 
 const circuitBreakerDefaults = {
-  timeout: '0.5s',
-  maxFailures: 5,
+  timeout: 250,
```

Comment:
Are you 100% sure this works? I guess that this is past to `ms` behind scenes, and if it is past directly then the integer is going to be converted to `250ms` string: https://www.npmjs.com/package/ms#convert-from-milliseconds

Reply:
https://github.com/auth0/node-baas3/blob/master/client.js#L9

We tested this for sure.